backend/src/main/kotlin/com/group7/artshare/entity/RegisteredUser.kt

```diff
@@ -94,13 +94,27 @@ class RegisteredUser(
     )
     var bookmarkedOnlineGalleries: MutableSet<OnlineGallery> = mutableSetOf()
 
+    @ManyToMany(cascade = [CascadeType.PERSIST, CascadeType.MERGE])
+    @JoinTable(
+        name = "read_notifications",
+        joinColumns = [JoinColumn(name = "user_id")],
+        inverseJoinColumns = [JoinColumn(name = "notification_id")]
+    )
+    var readNotifications: MutableSet<Notification> = mutableSetOf()
+
+    @ManyToMany(cascade = [CascadeType.PERSIST, CascadeType.MERGE])
+    @JoinTable(
+        name = "unread_notifications",
+        joinColumns = [JoinColumn(name = "user_id")],
+        inverseJoinColumns = [JoinColumn(name = "notification_id")]
+    )
+    var unreadNotifications: MutableSet<Notification> = mutableSetOf()
 
     //TODO discussion post
     //TODO past reply past posts
-    //TODO read notifications
-    //TODO unread notifications
-    //TODO current bids
 
+    @OneToMany(orphanRemoval = true, cascade = [CascadeType.ALL])
+    var currentBids: List<Bid> = ArrayList()
```

Comment:
MutableList may be required.

Reply:
You are right, updated with the latest commit.